govuk-one-login/authentication-stubs
# AUT-3862: Record `sub` from `/authorize` request

## What

The `sub` field is sent to `/authorize` and we need to return it when the `/reverification` API is requested.

Here we pass it though the `/authorize` form as a hidden field and store it in the Reverification table as the return object.

This object will eventually be returned by the `/reverification` API.

## How to review

1. Code Review
2. `npm run encryptSampleRequest`
3. Take the returned string and use it to get to `http://localhost:3000/authorize?request=[string-here]`
4. Open the browser developer tools and see the unique `sub` value in a hidden form field 
5. Click continue
6. See in the `Reverification` table that the same `sub` value is present in a record.
